include/navigation/infomax.h

```diff
@@ -61,29 +60,27 @@ class InfoMax : public VisualNavigationBase
     {}
 
     //------------------------------------------------------------------------
-    // VisualNavigationBase virtuals
+    // Public API
     //------------------------------------------------------------------------
-    virtual void train(const cv::Mat &image) override
+    void train(const cv::Mat &image, const ImgProc::Mask& = ImgProc::Mask{})
```

Comment:
It's a bit weird to have default values for unused args, but it's harmless and we'll change this when we fix #221 anyway.

Reply:
yeah - I left that looking like a loose end semi on purpose